images/singleuser/install-pwb

```diff
@@ -4,9 +4,6 @@ set -e
 # Remove 'Pillow' from requirements.txt
 sed -e '/Pillow/d' /srv/paws/pwb/requirements.txt > /tmp/requirements.txt
 
-# To prevent breakage on older python
-/srv/paws/bin/pip install --no-cache-dir -U pip setuptools wheel
```

Comment:
I'm not sure about removing this. In the Dockerfile we are running mostly system pip. This is the user-editable venv for pywikibot created at https://github.com/toolforge/paws/blob/master/images/singleuser/Dockerfile#L146
Since this PR will update the system pip and setuptools higher up in the dockerfile, they get symlinked into the venv, so I guess it doesn't matter and this is probably a more complete setup.

Reply:
Never mind, the path is altered and there is no real work on the system pip. Bleh.